src/core/card/__mocks__/card.mock.ts

```diff
@@ -2,7 +2,6 @@ const card = {
   imageUrl:
     "https://api.hel.fi/linkedevents-test/media/images/a-sip_netin_tapahtumakuva.1080x648_T3brYn4.jpg",
   subTitle: "12.12.2021",
-  target: "_blank",
```

Comment:
I think this was here in purpose, because it really can be configured from the Headless CMS. It's an feature that should be taken care of.